tests/testComposite.py

```diff
@@ -189,9 +187,7 @@ class TestGenericAssembler(unittest.TestCase):
     """A test case for the generic assembler feature of composite datasets."""
 
     def setUp(self):
-        packageDir = getPackageDir('obs_base')
-        self.testData = os.path.join(packageDir, 'tests', 'genericAssembler')
-        self.tearDown()
+        self.testData = tempfile.mkdtemp(dir=ROOT, prefix='genericAssembler-')
```

Comment:
it would be more consistent with other tests to use "TestCompositeTestCase" (the test class name) as the prefix.